.travis.yml

```diff
@@ -0,0 +1,30 @@
+language: octave
+
+os:
+  - linux
+  - osx
+
+compiler:
+  - gcc
+  - clang
+
+addons:
+  apt:
+    packages:
+      - mplayer
+    
+before install:
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew tap homebrew/science; fi
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew update; fi
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew install mplayer fontconfig ftgl; fi
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew install octave --with-gui; fi
+  - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then sudo apt-get -qq update; fi
+  - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then wget -O- http://neuro.debian.net/lists/xenial.us-nh.full | sudo tee /etc/apt/sources.list.d/neurodebian.sources.list; fi
+  - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then sudo apt-key adv --recv-keys --keyserver hkp://pgp.mit.edu:80 0xA5D32F012649A5A9; fi
+  - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then sudo apt-get -qq update; fi
+  - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then sudo apt-get install -y mplayer fontconfig; fi
+  - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then sudo apt-get install -y octave; fi
+
+script:
+  - cd Psychtoolbox
+  - yes|octave --eval 'SetupPsychtoolbox.m'
```

Comment:
FWIW, in neurodebian I already run some of the tests and scripts which do not require hardware:  have a peak at my abomination:  https://github.com/neurodebian/Psychtoolbox-3/blob/debian/debian/rules#L85

it would be cool if those (or some of those) could be excercised here... but probably after initial CI test of buildability works ;)